sumologic/resource_sumologic_slo.go

```diff
@@ -0,0 +1,416 @@
+				Required: true,
+			},
+			"version": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+			},
+			"created_at": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"created_by": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"modified_at": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"modified_by": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"parent_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"is_system": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
+			},
+			//"content_type": {
+			//	Type:     schema.TypeString,
+			//	Optional: true,
+			//	Default:  "Slo",
+			//	ExactlyOneOf: []string{
+			//		"Slo",
+			//	},
+			//},
+			//"type": {
+			//	Type:     schema.TypeString,
+			//	Optional: true,
+			//	Default:  "SlosLibraryFolder",
+			//	ExactlyOneOf: []string{
+			//		"SlosLibrarySlo",
+			//		"SlosLibrarySloUpdate",
+			//	},
+			//},
+			"signal_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"compliance": {
+				Type:     schema.TypeList,
+				Required: true,
+				//MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"compliance_type": {
+							Type:     schema.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"Rolling",
+								"Calendar",
+							}, false),
+						},
+						"target": {
+							Type:         schema.TypeFloat,
+							Required:     true,
+							ValidateFunc: validation.FloatBetween(0, 100),
+						},
+						"timezone": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"size": {
+							Type:     schema.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"1d", "2d", "3d", "4d", "5d", "6d", "7d", "8d", "9d", "10d", "11d", "12d", "13d", "14d",
+							}, false),
+						},
+					},
+				},
+			},
+			"indicator": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"evaluation_type": {
+							Type:     schema.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"Window",
+								"Request",
+							}, false),
+						},
+						"queries": {
+							Type:     schema.TypeList,
+							Required: true,
+							MinItems: 1,
+							MaxItems: 2,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"query_group_type": {
+										Type:     schema.TypeString,
+										Required: true,
+										ValidateFunc: validation.StringInSlice([]string{
+											"Successful", "Unsuccessful", "Total", "Threshold",
+										}, false),
+									},
+									"query_group": {
```

Comment:
We can add validation for ensuring that there are at max 6 queries specified here.